### ISSUE TITLE
Add overlay to header text to decrease instances where text lacks sufficient contrast

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-aldine#582]
Text in the header (h1.entry-title and p.entry-description) are often rendered illegible or given insfficient contrast if the network administrators select the wrong combination of featured color and header image. We should consider adding an overlay behind the text to increase the likelihood of text meeting contrast against any image (as we do for each of the text blocks displayed on the Aldine homepage).  

Several clients currently do this via custom CSS, in case a reference implementation would be useful?
